specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2015-04-08/cosmos-db.json

```diff
@@ -4325,6 +4325,14 @@
             "$ref": "#/definitions/Location"
           }
         },
+        "locations": {
```

Comment:
what is this locations property?  Adding writable properties to GA api-version is not allowed. It must be done in a new api-version.

Reply:
this property has always been in our response, but we did not originally add it to the swagger. I am adding it now because of [this issue](https://github.com/Azure/azure-sdk-for-python/issues/6823). Should I only add it to the new API version that has not had an SDK generated yet? In the current swagger, "locations" is present in the request body, but not the response (even though we do return it in the API).

Reply:
@ravbhatnagar, can you please advise on whether I should add this property to the old api version or only the new version?

Reply:
I think its best to add in the new api-versions.